tests/cupyx_tests/scipy_tests/special_tests/test_beta.py

```diff
@@ -98,20 +103,20 @@ def test_arange(self, xp, scp, dtype, function):
         # return scp.special.betainc(a, b, x)
         return func(a, b, x)
 
-    @pytest.mark.parametrize('function', ['betainc', 'betaincinv'])
-    @testing.for_float_dtypes()
-    @testing.numpy_cupy_allclose(atol=1e-5, rtol=1e-5, scipy_name='scp')
-    def test_linspace(self, xp, scp, dtype, function):
-        import scipy.special  # NOQA
-
-        func = getattr(scp.special, function)
-        # TODO: Some choices of start/stop value can give mismatched location
-        #       of +inf or -inf values.
-        tmp = xp.linspace(-20, 21, 10, dtype=dtype)
-        a = tmp[:, xp.newaxis, xp.newaxis]
-        b = tmp[xp.newaxis, :, xp.newaxis]
-        x = xp.linspace(0, 1, 5, dtype=dtype)[xp.newaxis, xp.newaxis, :]
-        return func(a, b, x)
+    # @pytest.mark.parametrize('function', ['betainc', 'betaincinv'])
+    # @testing.for_float_dtypes()
+    # @testing.numpy_cupy_allclose(atol=1e-5, rtol=1e-5, scipy_name='scp')
+    # def test_linspace(self, xp, scp, dtype, function):
+    #     import scipy.special  # NOQA
+
+    #     func = getattr(scp.special, function)
+    #     # TODO: Some choices of start/stop value can give mismatched location
+    #     #       of +inf or -inf values.
+    #     tmp = xp.linspace(-20, 21, 10, dtype=dtype)
+    #     a = tmp[:, xp.newaxis, xp.newaxis]
+    #     b = tmp[xp.newaxis, :, xp.newaxis]
+    #     x = xp.linspace(0, 1, 5, dtype=dtype)[xp.newaxis, xp.newaxis, :]
+    #     return func(a, b, x)
```

Comment:
Looks like the last run passed. Is the plan to uncomment this one again or restrict the HIP version here as well?

Reply:
this one was unrelated, just a leftover of my test combining runs :D